producer/main.py

```diff
@@ -45,3 +47,30 @@ def produce(pg_host, pg_port, pg_database, pg_user, pg_password, pg_replication_
 
     p.connect()
     p.start_consuming()
+
+
+
+@click.command()
+@click.option('--db_configs', default=lambda: os.environ.get('DB_CONFIGS', None), required=True, help='DB Configs')
```

Comment:
For multiple dbs this looks a little dirty
I would prefer that we get all this config from a `yaml` file.

Eg. what if for different producer I want different settings altogether